### ISSUE TITLE
Drive Motor Disable Toggle

### ISSUE THREAD

**Issue by CaiB** [huskyroboticsteam/Orpheus#74]
Could we please implement a toggle button in the main GUI that simply disables the drive motors while it is enabled? I'd appreciate having it there to try and prevent any accidents from happening while the science station is drilling into the ground, because if someone hits a controller accidentally during that, we're going to have a really bad time.

**Comment by Obbay2** [comment]
Yep, that would be a bad time! We will see what we can do about this issue, however since our networking was written rather hastily it may be more time intensive to make this change than is worth.
